### PR TITLE
Adds limit_eps API configuration tests

### PR DESCRIPTION
**Pull request by RamosFe**
|Related issue|
|-------------|
|     https://github.com/wazuh/wazuh/issues/20854        |

## Description

Adds multiple test cases for the `limit_eps` API configuration test.

### CODE REVIEW

File: tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import pytest
```
Comment (GGP1) [inline tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py:1]: We should add the copyright at the top of this file. Also, I'm not sure if it is a requirement but the same docstring generally contains the test description, components, suite, targets, daemons, etc. See [test_upload_configuration.py](https://github.com/wazuh/wazuh/blob/cdc44097c91876950185faddbff7e6902a0ee5e1/tests/integration/test_api/test_config/test_upload_configuration/test_upload_configuration.py) for example. We should check if it must be added and do it accordingly.

File: tests/integration/test_api/test_config/test_limit_eps/data/test_cases/limit_eps_config.yaml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            <eps>
+              <maximum>500</maximum>
+              <timeframe>50</timeframe>
+            </eps>
+          </limits>
+        </global>
+      </ossec_config>
+
+- name: UPLOAD_ONE_BLOCK_CONFIG_ALLOWED
+  description: Upload ones block configuration
```
Comment (GGP1) [inline tests/integration/test_api/test_config/test_limit_eps/data/test_cases/limit_eps_config.yaml:27]:
```suggestion
  description: Upload one block configuration
```

File: tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+# Marks
+pytestmark = pytest.mark.server
+
+# Variables
+# Used by add_configuration to select the target configuration file
+configuration_type = CONFIGURATION_TYPES[0]
+
+# Paths
+test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'limit_eps_api_config.yaml')
+test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'limit_eps_config.yaml')
```
Comment (GGP1) [inline tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py:20]: Again, not sure if this is a must. But all the other tests configurations and cases start with `configuration` and `cases` respectively. We should check whether this is necessary and apply the adjustments.

```suggestion
test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'configuration_limit_eps_configuration.yaml')
test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_limit_eps_configuration.yaml')
```

**Review by GGP1 — CHANGES_REQUESTED** [review]

File: tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py (hunk 1 above)
Comment (RamosFe) [reply tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py:49]: Added in commit 55c3deb54b1205546430149e7a80b348833da665.

File: tests/integration/test_api/test_config/test_limit_eps/data/test_cases/limit_eps_config.yaml (hunk 2 above)
Comment (RamosFe) [reply tests/integration/test_api/test_config/test_limit_eps/data/test_cases/limit_eps_config.yaml]: Changed in commit 55c3deb54b1205546430149e7a80b348833da665.

File: tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py (hunk 3 above)
Comment (RamosFe) [reply tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py]: Changed in commit 55c3deb54b1205546430149e7a80b348833da665.

**Review by GGP1 — APPROVED** [review]
LGTM!

> The API tier 0 and 1 tests fail because of https://github.com/wazuh/wazuh/issues/21402. The fix is not yet in master.

**Review by Selutario — CHANGES_REQUESTED** [review]
GJ @RamosFe. The failing API IT check should be fixed now that this PR was merged:
- https://github.com/wazuh/wazuh/pull/21638

Try rebasing your branch.

**Review by Selutario — APPROVED** [review]
LGTM.

**Pull request merged**